docs/resources/Guild.md

```diff
@@ -376,8 +376,8 @@ Modify attributes of a [guild member](#DOCS_RESOURCES_GUILD/guild-member-object)
 |-------|------|-------------|------------|
 | nick | string | value to set users nickname to | MANAGE_NICKNAMES |
 | roles | array of snowflakes | array of role ids the member is assigned | MANAGE_ROLES |
-| mute | boolean | whether the user is muted in voice channels | MUTE_MEMBERS |
-| deaf | boolean | whether the user is deafened in voice channels | DEAFEN_MEMBERS |
+| mute | ?boolean | whether the user is muted in voice channels. Will throw a 400 if the user is not in a voice channel | MUTE_MEMBERS |
+| deaf | ?boolean | whether the user is deafened in voice channels. Will throw a 400 if the user is not in a voice channel | DEAFEN_MEMBERS |
```

Comment:
Optional fields are denoted with a postfix `?` on the field name. [See Reference](https://discordapp.com/developers/docs/reference#nullable-and-optional-resource-fields-example-nullable-and-optional-fields)

Reply:
Seems redundant to mark these as optional when the info box above states `All parameters to this endpoint are optional`.

Reply:
Fair point. Is `All parameters to this endpoint are optional` standard? If not, following the standard of marking fields as optional should be used.

Reply:
Its present for a few endpoints but talking about a standard with these docs is difficult

Reply:
>Is All parameters to this endpoint are optional standard?
For PATCH requests, yes, this is the usual case (can't really think of any exceptions to this either).